automatic/selenium.powershell/update.ps1

```diff
@@ -0,0 +1,74 @@
+﻿param(
+    [string]$ModuleName = 'Selenium'
+)
+Import-Module Chocolatey-AU
+
+function global:au_SearchReplace {
+  @{
+    ".\legal\VERIFICATION.txt"      = @{
+      # Replace the module version on the Save-Module line
+      "(?i)(^\s*\-\s*Save\-Module.*\-Version\s+).*" = "`${1}$($Latest.ModuleVersion)"
+
+      # Attempt to remove all existing checksums
+      "(?i)(^\s*file\s*name\:).*"         = ""
+      "(?i)(^\s*checksum\s*type\:).*"     = ""
+      "(?i)(^\s*checksum(32)?\:).*"       = ""
```

Comment:
These, or at least some of these are wrong. The AppVeyor build is failing due to not being able to replace this regex values.

They also should not be empty, please fill out this section.

Reply:
To be clear, as in the comment "Attempt to remove all existing checksums", this is removing the lines. The emptiness is a feature.

Reply:
The failure was definitely because of the lack of existing checksums. I added one of them back for the PR, and it is working locally for me (I'm not entirely sure why it worked before for me... It's possible I might not have had a fully clean working directory, or was working in pwsh)

Reply:
Then it sounds like a different approach is needed, something to consider at least.

Maybe keep a seperate file as the template of this, then during update that file is read in, any additional information is appended to the file and then it writes/overwrites the VERIFICATION.txt file, that should be done in the AfterUpdate method though in that case, and then remove this regexes.